Unbox.swift

```diff
@@ -696,12 +691,64 @@ private extension Unboxer {
         
         return unboxed
     }
+
+    /// Unbox a JSON dictionary into a model `T`, while optionally using a contextual object. Throws `UnboxError`.
+    static func performUnboxingWithDictionary<T: Unboxable>(dictionary: UnboxableDictionary, context: Any? = nil) throws -> T {
+        return try Unboxer(dictionary: dictionary, context: context, allowInvalidElements: false).performUnboxing()
+    }
+
+    /// Unbox an array of JSON dictionaries into an array of `T`, while optionally using a contextual object. Throws `UnboxError`.
+    static func performUnboxingWithDictionary<T: Unboxable>(dictionaries: [UnboxableDictionary], context: Any? = nil, allowInvalidElements: Bool = false) throws -> [T] {
+        return allowInvalidElements
```

Comment:
To make this line more readable (it wraps a lot in the GH interface), can you break it out into an `if` statement?